python/bin/suricatasc

```diff
@@ -1,14 +1,15 @@
-#!/usr/bin/python
-# Copyright(C) 2013-2020 Open Information Security Foundation
-
+#! /usr/bin/env python
+#
+# Copyright(C) 2013-2022 Open Information Security Foundation
+#
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation, version 2 of the License.
 #
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
+# GNU General Public License for mo re details.
```

Comment:
nit: typo

Reply:
fixup in staging

Reply:
> fixup in staging

Did this make it into any staging branch?  We'll want this in 7.0 for sure.  I think we'll start to see breakage in the upcoming releases of distributions.

Reply:
Hmm.. So probably want it for 6.0.x as well.

Reply:
I guess not. Don't remember what happened.